### PR TITLE
Update moose

### PR DESCRIPTION
**Pull request by mariusSincovici**
Update
* tracker to v2.0.1 (libmoose v9.0.0)
* worker to v8.2.0(libmoose v8.2.0)

### CODE REVIEW

File: events/moose/notify.go [hunk 1]
```diff
@@ -5,57 +5,62 @@ package moose
...
-	responseCode int,
+	requestFields string,
+	limits string,
+	offset string,
 	responseSummary string,
-	transferProtocol string,
-) uint
+) uint32
+
+// (	hostname string,
```
Comment (keliramu) [inline events/moose/notify.go:26]: Is this some leftovers and maybe should be removed?

File: go.mod [hunk 2]
```diff
@@ -30,19 +30,19 @@ require (
...
 	golang.org/x/exp v0.0.0-20230310171629-522b1b587ee0
 	golang.org/x/mod v0.11.0
+	golang.org/x/net v0.20.0
 	golang.org/x/oauth2 v0.7.0
 	golang.org/x/sys v0.16.0
 	golang.org/x/term v0.16.0
 	golang.zx2c4.com/wireguard v0.0.0-20230313165553-0ad14a89f5f9
-	google.golang.org/api v0.114.0
-	google.golang.org/grpc v1.56.3
+	google.golang.org/api v0.47.0
```
Comment (keliramu) [inline go.mod:38]: Is downgrade really needed?

File: go.mod [hunk 3]
```diff
@@ -55,7 +55,7 @@ require (
 	github.com/golang/protobuf v1.5.3 // indirect
 	github.com/google/pprof v0.0.0-20210407192527-94a9f03dee38 // indirect
 	github.com/googleapis/enterprise-certificate-proxy v0.2.3 // indirect
-	github.com/googleapis/gax-go/v2 v2.7.1 // indirect
+	github.com/googleapis/gax-go/v2 v2.0.5 // indirect
```
Comment (keliramu) [inline go.mod:58]: Also downgrade - maybe not needed?

File: go.mod (hunk 2 above)
Comment (mariusSincovici) [reply go.mod]: needed to run `go mod tidy` and fixed all the dependencies.

**Review by Savolro — APPROVED** [review]

**Pull request merged**